src/libraries/Microsoft.Extensions.Configuration/tests/ConfigurationTest.cs

```diff
@@ -889,5 +889,28 @@ public void NullSectionDoesNotExist()
             // Assert
             Assert.False(sectionExists);
         }
+
+        internal class NullReloadTokenConfigSource : IConfigurationSource, IConfigurationProvider
+        {
+            public IEnumerable<string> GetChildKeys(IEnumerable<string> earlierKeys, string parentPath) => throw new NotImplementedException();
+            public Primitives.IChangeToken GetReloadToken() => null;
+            public void Load() { }
+            public void Set(string key, string value) => throw new NotImplementedException();
+            public bool TryGet(string key, out string value) => throw new NotImplementedException();
+            public IConfigurationProvider Build(IConfigurationBuilder builder) => this;
+        }
+
+        [Fact]
+        public void ProviderWithNullReloadToken()
+        {
+            // Arrange
+            var builder = new ConfigurationBuilder();
+            builder.Add(new NullReloadTokenConfigSource());
+
+            // Act
+            var config = builder.Build();
+
+            // Assert
+        }
```

Comment:
Can we at least assert that `Assert.IsNotNull(config)`?

Reply:
Done